src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/gemm/gemm_kernel_ref.cpp

```diff
@@ -12,16 +12,14 @@ ParamsKey GemmKernelRef::GetSupportedKey() const {
     k.EnableInputDataType(Datatype::F32);
     k.EnableInputDataType(Datatype::INT8);
     k.EnableInputDataType(Datatype::UINT8);
+    k.EnableInputDataType(Datatype::INT32);
```

Comment:
are there I32 tests?

Reply:
yes, look for data_types::f16, data_types::f32 , data_types::i8, data_types::u8, data_types::i32 line in gemm_gpu_test.cpp file